### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by Abdona**
Using Bootstrap5 to style the webpage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            </path>
+          </g>
+        </svg>
+      </div>
```
Comment (MuriloRoque) [inline index.html]: - [x] This logo should be a link.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+  <title>Newsweek</title>
+  <link rel="stylesheet" href="Assests/css/bootstrap.css" />
+  <link rel="stylesheet" href="Assests/scss/main.css" />
+  <script src="js/bootstrap.js"></script>
+</head>
+
+<body class="arial-font">
+  <header>
+    <div class="container-fluid justify-content-between align-items-center p-3 d-flex back-ground-color">
+        <p class="d-none d-lg-block">Wed, Jan 20, 2021</p>
```
Comment (MuriloRoque) [inline index.html]: - [x] The color of the date is not the same as the original, please fix it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        </svg>
+      </div>
+      <div class="d-flex justify-content-between align-items-center ">
+        <span class="d-lg-none"></span>
+        <span class="d-lg-none"></span>
+        <span class="d-lg-none"></span>
+        <a href="#" class="d-none d-lg-inline-block">LOGIN</a>
+        <button class="btn btn-warning roboto-font rounded-pill">
+          SUBSCRIBE &rsaquo;
+        </button>
```
Comment (MuriloRoque) [inline index.html]: - [x] The fonts and colors of these are also not the same as the original, please try achieving a result as close to the original as possible :smile:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">Experts<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">Vantage</a>
+          </li>
+          <li class="fs-6 mt-2 mx-2 text-black-50 fw-light">Search</li>
```
Comment (MuriloRoque) [inline index.html]: - [x] You are missing the search icon here, can you please add a magnifying glass just so it is like the original? 
- [x] This should also be wrapped in a `form` with an `input` tag, so I could type anything I need to search on your webpage.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+            </path>
+          </g>
+        </svg><span>&copy; 2021 NEWSWEEK DIGITAL LLC
```
Comment (MuriloRoque) [inline index.html]: - [x] This one should also be a link :smile:

File: index.html [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+        </span></div>
+      <div class="float-end ">fonts</div>
```
Comment (MuriloRoque) [inline index.html]: - [x] You are missing the social media icons here, please add them. And remember that they should also be links!

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @Abdona :wave: 

### You have done an amazing job on this project, I can see that you really put any effort into it, as it's almost close to the original, I must congratulate you :clap: :clap: :clap: 

### We still need to fix somethings before moving on, alright? Can you please check my comments in your code below?

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

File: index.html (hunk 6 above)
Comment (Abdona) [reply index.html]: I fixed and added all the changes requested.
Thank you for your review.

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔**

Hello @Abdona 
Great work making the requested changes. 
You can move to another milestone👌👌👍👍

**Pull request merged**